src/app/src/pages/redteam/report/page.tsx

```diff
@@ -16,7 +16,7 @@ export default function ReportPage() {
   const searchParams = new URLSearchParams(window.location.search);
   const evalId = searchParams.get('evalId');
   usePageMeta({
-    title: evalId ? 'Red team report' : 'Red team reports',
+    title: evalId ? 'Red team report' : 'Red Team Vulnerability Reports',
```

Comment:
why?

Reply:
The eval results CTA uses "Vulnerability Report" as its label. "Red Team Vulnerability Reports" makes sense with "Vulnerability Report" as the truncated form but "Red Team Reports" does not.

Reply:
I don't understand what the conditional is doing here